pandas_gbq/gbq.py

```diff
@@ -146,12 +148,17 @@ def __init__(self, project_id, reauth=False, verbose=False,
         _test_google_api_imports()
         self.project_id = project_id
         self.reauth = reauth
-        self.verbose = verbose
```

Comment:
ok, can you change the default above to ``None``. I think will have to change this in pandas as well (its currently ``verbose=True``).